StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1009ClosingParenthesisMustBeSpacedCorrectly.cs

```diff
@@ -87,6 +87,13 @@ private static void HandleCloseParenToken(SyntaxTreeAnalysisContext context, Syn
             switch (nextToken.Kind())
             {
             case SyntaxKind.OpenParenToken:
+                // Allow a space between an open and a close paren when:
+                // - they are part of an if statement
+                // - they are on the same line
+                // - the open paren is part of a parenthesized expression
```

Comment:
:bulb: Can also happen for a tuple expression. It would be good to go ahead and cover this case.

```csharp
if (true) (1, 2).ToString();
```